### PR TITLE
Adding info in Requisition Lists doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request describes common feature information for better understanding

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

 https://docs.magento.com/user-guide/configuration/customers/requisition-lists.html

## Links to Magento source code or PRs

https://github.com/magento/partners-magento2b2b/blob/develop/app/code/Magento/RequisitionList/README.md

See Contribution guidelines (https://github.com/magento/merchdocs/blob/master/.github/CONTRIBUTING.md) and wiki (https://github.com/magento/merchdocs/wiki) for more information.

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
Nice work.

**Comment by leslietilling** [comment]
@dnahirna  Thanks for helping with docs. I have a question about calling this a "module". Does this require a separate install on top of the B2B extension? If so, is there information in devdocs for installation?  

This page is a config reference page and it might make more sense to put this info into the main part of the guide. I can add a new page/section into our B2B config overview/procedures and move it there.

File: src/configuration/customers/requisition-lists.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ b2b_only: true
 title: Requisition Lists
 ---
 
+The Magento_RequisitionList module allows a customer to create multiple lists of frequently-purchased items and use those lists for order placement.
```
Comment (atwixfirster) [inline src/configuration/customers/requisition-lists.md:6]: @dnahirna , could you please replace `The Magento_RequisitionList module` with `The feature`.

File: src/configuration/customers/requisition-lists.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ b2b_only: true
...
+This feature is available for both logged-in users and guests.
```
Comment (atwixfirster) [inline src/configuration/customers/requisition-lists.md:7]:
```
It is available for both logged-in users and guests.
```

File: src/configuration/customers/requisition-lists.md [hunk 3]
```diff
@@ -13,3 +25,7 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Cust
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
 |Number of Requisition Lists|Store View|Determines the maximum number of requisition lists that can be created per customer account. The minimum number is 2, and the maximum number is 999.|
+
+The module does not create any backward incompatible changes. It can be deactivated and uninstalled in any time.
```
Comment (atwixfirster) [inline src/configuration/customers/requisition-lists.md:29]: Please remove this sentense.

THank you!

**Comment by atwixfirster** [comment]
> @dnahirna Thanks for helping with docs. I have a question about calling this a "module". Does this require a separate install on top of the B2B extension? If so, is there information in devdocs for installation?
> 
> This page is a config reference page and it might make more sense to put this info into the main part of the guide. I can add a new page/section into our B2B config overview/procedures and move it there.

Hi, @leslietilling !

The main idea of this PR: provide information about Requisition List feature, its benefits. Also, please let us know there is the best place to put this information.

Thank you very MUCH as always!

File: src/configuration/customers/requisition-lists.md (hunk 1 above)
Comment (dnahirna) [reply src/configuration/customers/requisition-lists.md]: done

File: src/configuration/customers/requisition-lists.md (hunk 2 above)
Comment (dnahirna) [reply src/configuration/customers/requisition-lists.md]: done

File: src/configuration/customers/requisition-lists.md (hunk 3 above)
Comment (dnahirna) [reply src/configuration/customers/requisition-lists.md]: done

**Comment by leslietilling** [comment]
I will merge to an integration branch and make the changes before we merge to master.

**Pull request merged**